src/mozilla_sec_eia/ex_21/train_extractor.py

```diff
@@ -140,6 +144,7 @@ def log_model(finetuned_model: Trainer):
 
 def load_model():
     """Load fine-tuned model from mlflow artifacts."""
+    # TODO: want the ability to give load_model a model path?
     initialize_mlflow()
     return mlflow.transformers.load_model(
         "models:/layoutlm_extractor/1", return_type="components"
```

Comment:
Is it possible to choose which model this loads? How does MLFlow store the models? How does it handle versioning? Is it possible to pass a path in so that you can choose which version it loads?

Reply:
Yeah I'm going to do a little more research on model tracking/versioning to try and come up with a good system for loading the desired version